EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -128,6 +150,8 @@ extension MediaPlayerView {
         @Published var playerState: PlayerState = .loading
         @Published var inLoopEnabled: Bool = false
         @Published var isShowingActionButtons = true
+        @Published var playerDuration: Float64 = 1.0
+        @Published var playerCurrentTime: Float64 = 0.0
```

Comment:
Any reason for not using `TimeInterval` here? Btw, `Float64` sounds like a `Double` to me haha